src/display/editor/toolbar.js

```diff
@@ -163,7 +163,7 @@ class EditorToolbar {
 
   async addButton(name, tool) {
     switch (name) {
-      case "colorPicker":
+      case "highlightColorPicker":
```

Comment:
This ties in to the renaming comment above, but this case seems to be identical to the one of `colorPicker` below. Can we collapse them into one to have a generic color picker addition call for all annotation types, and let the annotation types themselves determine (via `toolbarButtons`) if they provide the basic or the regular color picker since that seems simpler overall?

Reply:
Yep I just forgot to remove that stuff.